.goreleaser.yaml

```diff
@@ -1,31 +1,39 @@
-# .goreleaser.yaml
 builds:
-- skip: true # skip build for now since we have another mechanism to build all kubestellar binaries
-  id: kubestellar-syncer
-  main: ./cmd/syncer
-  binary: kubestellar-syncer
+- id: "controller-manager"
+  main: ./cmd/controller-manager
+  binary: bin/controller-manager
+  ldflags:
+  - "{{ .Env.LDFLAGS }}"
   goos:
   - linux
-  - darwin
   goarch:
   - amd64
   - arm64
-  - s390x
+  ignore:
+  - goos: darwin
+    goarch: ppc64le
+  env:
+  - CGO_ENABLED=0
+kos:           
+  - repository: ghcr.io/kubestellar/kubestellar/controller-manager
+    main: ./cmd/controller-manager
+    build: controller-manager
+    tags:
+    - '{{.Version}}'
+    - latest
+    bare: true
+    preserve_import_paths: false
+    ldflags:
+    - "{{ .Env.LDFLAGS }}"
+    platforms:
+    - linux/amd64
+    - linux/arm64
+    env:
+    - CGO_ENABLED=0
```

Comment:
Didn't we just say this above?

Reply:
If you mean the example at https://goreleaser.com/customization/ko/#example , that does not show that CGO_ENABLED MUST only appear in one place. Is there some doc that says specifying it in two places is forbidden?

Anyway, it looks like we are allowed to specify it in only one place and that is what you have now, so that's good.